data/remnant missions.txt

```diff
@@ -1506,3 +1510,150 @@ mission "Remnant: Broken Jump Drive job"
+			`	You grab the tendril and store it a sealed container. While the attack spoiled the beautiful moment, at least you have a sample and some recordings that might be of interest to Plume.`
+				flee
+			label holdground
+			`	Despite the feelings filling your mind, you resolve yourself to face down the void sprite without triggering your shields. Time seems to slow down as the sprite rushes up to you and its leading tentacle reaches out and touches you. As you brace for an impact you realize that the sprite has managed to bring itself to a complete stop and the tentacle has telescoped in on itself slightly. You can see that it is touching you, but you didn't even feel the impact. You and the void sprite stand frozen for a moment, and the mind-numbing paralysis snaps off, replaced by a feeling of weightlessness. Moments later the void sprite pulls back and pulses skyward, spiraling around the <ship> before disappearing into the swirling clouds.`
+			`	As you re-enter the ship, you notice that your suit is covered in some kind of translucent slime. You scrape off as much as you can into a sealed container, but your pressure suit appears to be permanently stained an iridescent purplish color as a result. You resolve to go find Plume, since he will almost certainly find this interesting.`
+				accept
+	on accept
+		fail "Remnant: Face to Maw 1b"
+	on complete
+		conversation
+			`When you walk into Plume's lab on Aventine you find him buried in some kind of holographic projection of your egg returning mission. You can see that he appears to be watching the video in slow-motion as he taps out notes on his datapad. After a few minutes he looks up and see you standing in the entryway. He quickly pulls himself out of the projection and hits what you assume must be the pause button.`
+			`	"Captain <first>! What a pleasant surprise," he trills. "If you are looking for more work, I don't have anything for you right now."`
+			`	"Actually, I have something for you," you reply. "I visited Nenia and had an interesting encounter with a void sprite. I came away from it with some more footage and a container of slime." He eagerly accepts the datacube and the container from you. He slides the container into some kind of machine that immediately starts whirring, and slots the datacube into his console. Moments later his holographic projection shifts to the new footage. He checks the time stamps, then chants "These will take me a little while to analyze. I will meet you in the spaceport in a few hours to discuss my preliminary findings."`
```

Comment:
This feels a little off to me. I think that in situations like this, typically the conversation refers to you speaking without direct dialog. E.g. in this scenario, it may instead be `You tell Plume about how you visited Nenia and had an interesting encounter with a void sprite...`

Reply:
Rewrote this section to remove the direct dialogue on the part of the player.